### PR TITLE
chore: api and version polish

### PR DESCRIPTION
**Pull request by mfw78**
# Description
This PR applies some fixes / feedback from previous PRs.

# Changes

- [x] All versioning within the repository now refers to `package.json`.
- [x] `/about` endpoint moved to `/version` endpoint to align with services repo.
- [x] DRY'd the `run` command.
- [x] Rest API object only created if it's intended to be used.

## How to test

1. Run `yarn ts-node ./src/index.ts run --rpc http://10.23.0.2:8545 --rpc http://10.23.0.2:8645 --deployment-block 17883049 --deployment-block 29389123 --page-size 10000` and observe synchronisation.

**NOTE**: Substitute your RPCs in the above.

## Related Issues

Related #89

### CODE REVIEW

File: src/commands/run.ts [hunk 1]
```diff
@@ -56,8 +54,20 @@ export async function run(options: RunOptions) {
...
 }
+
+/**
+ * Run actions required when stopping the watch-tower from run mode
+ * @param exitCode Exit code to return to the shell
+ */
+function stop(exitCode?: number) {
+  const log = getLogger("commands:stop");
+  log.info("Stopping Rest API server...");
+  ApiService.getInstance().stop();
```
Comment (anxolin) [inline src/commands/run.ts:68]: this is a promise. Shouldn't yuou await it? 

same as for the db service.
probably you should catch an error closing it.

Q: can the two services be closed in paralle? then you could do a promise. If so, probably you could use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled

Comment (mfw78) [reply src/commands/run.ts:68]: Great catch. Silly me. Will modify and push with the Promise.allSettled.

File: src/types/index.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export interface RunOptions extends WatchtowerOptions {
   silent: boolean;
   slackWebhook?: string;
   oneShot: boolean;
-  enableApi: boolean;
+  disableApi: boolean;
   apiPort: number;
```
[the remark this answers is not in the document]
Comment (mfw78) [reply src/types/index.ts:21]: Agree it's not great for our reasoning as developers, however IMO from a _user's_ perspective, it makes configuration easier to reason, which was my motivation on it.

File: src/utils/api.ts [hunk 3]
```diff
@@ -50,7 +55,7 @@ export class ApiService {
         }
         this.server = this.app.listen(this.port, () => {
           log.info(
-            `Starting Rest API server on port ${this.port}. See http://localhost:8080/api/about`
+            `Rest API server is running on port ${this.port}. See http://localhost:${this.port}/api/version`
```
[the remark this answers is not in the document]
Comment (mfw78) [reply src/utils/api.ts:58]: Hehe, no probs :+1:

**Pull request merged**